app/components/navbar/navbar_component.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="navbar">
+    <div class="logo"> 
+        <%= render "components/logo" %>
+    </div>
+    <div class="navbar__desktop">
+        <ul>
+            <%= nav_link("Home", '/') %>
+            <% resources.each do |resource| %>
+                <%= nav_link(resource[:front_matter][:title], resource[:path]) %>
+            <% end %>
+            <%= nav_link("Find an event near you", events_path) %>
+        </ul>
+    </div>
+        
+    <div class="navbar__mobile" data-controller="navigation">
+        <div class="navbar__mobile__buttons">
+            <a data-action="click->navigation#navToggle" href="javascript:void(0);" class="icon">
+                <div data-target="navigation.hamburger" id='hamburger' class="icon-close"></div>
+                <div data-target="navigation.label" id="navbar-label" class="icon-hamburger-label">Close</div>
+            </a>
+        </div>
+        <div data-target="navigation.links" id="navbar-mobile-links" class="navbar__mobile__links">
+            <ul>
+                <li><a href="/">Home</a></li>
+                <% resources.each do |resource| %>
+                    <li><a href="<%= resource[:path] %>"><%= resource[:front_matter][:title] %></a></li>
```

Comment:
Shoulld probably be using a `link_to` here rather than manually building the `a` element